src/OrchardCore.Modules/OrchardCore.Markdown/Drivers/MarkdownFieldDriver.cs

```diff
@@ -41,8 +41,9 @@ public override IDisplayResult Display(MarkdownField field, BuildFieldDisplayCon
                 model.PartFieldDefinition = context.PartFieldDefinition;
                 templateContext.SetValue("Model", model);
 
-                model.Markdown = await _liquidTemplatemanager.RenderAsync(field.Markdown, _htmlEncoder, templateContext);
-                model.Html = Markdig.Markdown.ToHtml(model.Markdown ?? "");
+                var markdown = await _liquidTemplatemanager.RenderAsync(field.Markdown, _htmlEncoder, templateContext);
```

Comment:
I instead want to change the other one to have only Markdown (no liquid) in the Markdown property. We also do this in the HtmlField and HtmlBodyPart to store the Html without liquid.

Reply:
Okay